src/runtime/interrupt/interrupt_gameboyadvance.go

```diff
@@ -8,9 +8,9 @@ import (
 )
 
 var (
-	regInterruptEnable       = (*volatile.Register16)(unsafe.Pointer(uintptr(0x4000200)))
-	regInterruptRequestFlags = (*volatile.Register16)(unsafe.Pointer(uintptr(0x4000202)))
-	regInterruptMasterEnable = (*volatile.Register16)(unsafe.Pointer(uintptr(0x4000208)))
+	regInterruptEnable        = (*volatile.Register16)(unsafe.Pointer(uintptr(0x4000200)))
+	regInterruptRequestFlags  = (*volatile.Register16)(unsafe.Pointer(uintptr(0x4000202)))
+	regInterruptPrimaryEnable = (*volatile.Register16)(unsafe.Pointer(uintptr(0x4000208)))
```

Comment:
The "master enable" bit controls all interrupts at once, like cli/sei on AVR for example. I don't think "primary" is the correct term to describe this.

I'm not sure what to call this. Maybe it's better to keep calling it "master enable", because that is what upstream documentation uses: https://www.akkit.org/info/gbatek.htm#gbainterruptcontrol. Leaving it shouldn't matter much as it's not exposed in an API.

Reply:
Maybe `regGlobalInterruptEnable` if we still wanted to change it?

Reply:
That's a good idea, thanks!

Reply:
Done.